docs/api/screen.md

```diff
@@ -124,7 +124,7 @@ Returns [`Display`](structures/display.md) - The display nearest the specified p
 Returns [`Display`](structures/display.md) - The display that most closely
 intersects the provided bounds.
 
-### `screen.screenToDipPoint(point)` _Windows_
+### `screen.screenToDipPoint(point)` _Windows_ _Linux_
```

Comment:
I believe docs need to highlight that method only works for x11 and wayland is yet to be implemented.